pkg/core/blockchain.go

```diff
@@ -130,6 +131,11 @@ type Blockchain struct {
 
 	contracts native.Contracts
 
+	validators      []util.Uint160
+	oracles         []util.Uint160
+	stateValidators []util.Uint160
```

Comment:
Do we need these lists?

Reply:
Fixed.

Reply:
They we like cache of addresses for cached public keys. I guess because these changes won't be too often, we can calculate them again.